services/core-web/src/components/mine/Permit/MinePermitTable.tsx

```diff
@@ -188,494 +195,537 @@ const renderPermitNo = (permit) => {
+      handleDeletePermit,
+      handleDeletePermitAmendment,
+      openAddPermitHistoricalAmendmentModal,
+      openEditSitePropertiesModal,
+    };
+  };
+
+  const transformChildRowData = (
+    amendment,
+    record,
+    amendmentNumber,
+    major_mine_ind,
+    openEditAmendmentModal,
+    handleDeletePermitAmendment,
+    handlePermitAmendmentIssueVC,
+    permitAmendmentTypeOptionsHash,
+    openViewConditionModal,
+    mineGuid
+  ) => ({
+    amendmentNumber,
+    isAmalgamated: amendment.permit_amendment_type_code === PERMIT_AMENDMENT_TYPES.amalgamated,
+    receivedDate: formatDate(amendment.received_date) || Strings.EMPTY_FIELD,
+    issueDate: formatDate(amendment.issue_date) || Strings.EMPTY_FIELD,
+    authorizationEndDate: formatDate(amendment.authorization_end_date) || Strings.EMPTY_FIELD,
+    description: amendment.description || Strings.EMPTY_FIELD,
+    isAssociatedWithNOWApplicationImportedToCore: "",
+    openEditAmendmentModal,
+    openViewConditionModal,
+    permit: record.permit,
+    documents: amendment.related_documents,
+    handleDeletePermitAmendment,
+    handlePermitAmendmentIssueVC,
+    finalApplicationPackage: finalApplicationPackage(amendment),
+    maps: amendment.now_application_documents?.filter(
+      (doc) => doc.now_application_document_sub_type_code === "MDO"
+    ),
+    permitAmendmentTypeOptionsHash,
+    permitAmendmentDocuments: amendment.related_documents.map((doc) => ({
+      key: doc.mine_report_submission_guid,
+      documentManagerGuid: doc.document_manager_guid,
+      filename: doc.document_name,
+    })),
+    mineGuid,
+    ...amendment,
+  });
+
+  const columns: ColumnsType<MinePermitTableItem> = [
+    {
+      title: "Permit No.",
+      dataIndex: "permitNo",
+      key: "permitNo",
+      render: (text, record) => <div title="Permit No.">{renderPermitNo(record.permit)}</div>,
+    },
+    {
+      title: "Status",
+      dataIndex: "status",
+      key: "status",
+      render: (text, record) => <div title="Status">{record.permitStatusOptionsHash[text]}</div>,
+    },
+    {
+      title: "Permittee",
+      dataIndex: "permittee",
+      key: "permittee",
+      render: (text) => <div title="Permittee">{text}</div>,
+    },
+    {
+      title: "First Issued",
+      dataIndex: "firstIssued",
+      key: "firstIssued",
+      render: (text) => <div title="First Issued">{text}</div>,
+    },
+    {
+      title: "Last Amended",
+      dataIndex: "lastAmended",
+      key: "lastAmended",
+      render: (text) => <div title="Last Amended">{text}</div>,
+    },
+    {
+      title: "",
+      dataIndex: "addEditButton",
+      key: "addEditButton",
+      align: "right",
+      render: (text, record) => {
+        const { id } = useParams<{ id: string }>();
+        const permitGuid = isPermit(record.permit)
+          ? record.permit.permit_guid
+          : record.permit?.permit_amendment_guid;
+        const menu = (
+          <Menu>
+            <Menu.Item key="0">
```

Comment:
We should not add in the deprecated `Menu.Item` syntax where it's not pre-existing.
This should get changed out for the `items` syntax which is present in a few places in our codebase. 

The `useParams` hook should be removed- I don't _think_ that `id` is being used, but if it is it'll have to be passed down.